index.html

```diff
@@ -0,0 +1,183 @@
+                <a href="#" class="list">Mypage</a>
+                <a href="#" class="list">Korean</a>
+              </div>
+          </div>
+             <div class="navbar-container">
+                <div class="logo"><img src="Capstone/Header/logo.png" alt="simmitlogo"></div>
+                <div ><img src="capstone/Header/close_mobile_menu.png" alt="close-icon" id="close-icon"></div>
+                  <div class="nav-list">
+                    <a href="about.html" class="nav-menu">About</a>
+                    <a href="#"class="nav-menu">Program</a>
+                    <a href="#"class="nav-menu">Join</a>
+                    <a href="#"class="nav-menu">Sponsor</a>
+                    <a href="#"class="nav-menu">News</a>
+                    <a href="index.html"class="nav-menu" id="cc">CC Campaign</a>
+                 </div>
+               </div>
+      </div>
+        
+    </header>
+    <!-- intro section -->
+   <section class="intro-container">
+     <div class="background-image">
+         <div class="menu"><img src="Capstone/Header/mobile_menu.png" alt="meu-btn" id="menu-btn"></div>
+         <div class="text-container ">
+            <p id="first-para">"Hello! Sharing World"</p>
+            <h1 id="h1">Creative Commons Global Summit 2015</h1>
+            <p id="second-para">Lorem ipsum dolor sit amet consectetur adipisicing elit. Beatae, exercitationem distinctio maxime nobis blanditiis laudantium, iure recusandae aliquid, quas ipsam cum eos consequatur esse voluptatum dolores unde.</p>
+            <p id="text-para">2022.6.27(MON) ~ 7(TUE)</p>
+            <p id="third-para">@ National Museam of Korea,Art center Nabi an More</p>
+         </div>
+      </div>
+    
+   </section >
+   <!-- main program -->
+   <section  class="program-container">
+    <h2>Main Program</h2>
+           <div  class="grid-container">
+               <div class="grid-box">
+                 <img src="Capstone/Icons/program_icon_01.png" alt="lecture" class="image-box">
+                 <h3 class="heading-box">Lecture</h3>
+                 <p class="box-para">Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+               </div>
+               <div class="grid-box">
+                 <img src="Capstone/Icons/program_icon_02.png" alt="lecture" class="image-box">
+                 <h3 class="heading-box">CC Exhibition</h3>
+                 <p class="box-para">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+               </div>
+               <div class="grid-box">
+                 <img src="Capstone/Icons/program_icon_03.png" alt="lecture" class="image-box">
+                 <h3 class="heading-box">Forum</h3>
+                 <p class="box-para">Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+               </div>
+               <div class="grid-box">
+                 <img src="Capstone/Icons/program_icon_04.png" alt="lecture" class="image-box">
```

Comment:
- [ ] Kindly follow the same height for the mobile version as shown in the attached loom video.